python/ray/tests/test_logging_2.py

```diff
@@ -477,6 +469,27 @@ def print_message(self):
     assert "(MyActor pid=" not in stderr
 
 
+def test_configure_both_structured_logging_and_lib_logging(shutdown_only):
+    """
+    Import `ray.data` to configure the `ray.data` logger. Then, configure the
+    `root` and `ray` loggers in `ray.init()`. Ensure that handlers are not reset.
+    """
+    script = """
+import ray
+import logging
+import ray.data
```

Comment:
Instead of relying ray.data, let's manually config a `ray.test` logger before `ray.init()`

Reply:
updated